universe/auto_syncer.go

```diff
@@ -313,6 +452,51 @@ func (f *FederationEnvoy) syncer() {
 				continue
 			}
 
+			// After we've synced with the federation, we'll
+			// attempt to push out any pending proofs that we
+			// haven't yet completed.
+			ctxFetchLog, cancelFetchLog := f.WithCtxQuitNoTimeout()
+			syncDirection := SyncDirectionPush
+			logEntries, err :=
```

Comment:
nit: extract `db := f.cfg.FederationDB` instead of breaking after the `:=` part for better readability?

Reply:
Sure no problem, will do. In a future PR I hope to break down each `case` in this `select` statement into three methods. This will simplify error handling (avoid the need to log everywhere rather than just return) and will save on indentation. I think it will also help with variable name shadowing.